src/index.html

```diff
@@ -11,6 +11,41 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="wrapper">
+      <form class="search-form search-form_large"
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <label class="search-label">
+          <div class="flex">
```

Comment:
Why do you need this additional `div` here? I think you can remove it.

Reply:
it makes sure that the input field does not go beyond the screen and becomes flexible

Reply:
![2023 02 02](https://user-images.githubusercontent.com/114246735/216289431-d091e968-1b9e-4012-8dcd-07c6d73b5b60.jpg)

Reply:
Not fixed.
You don't need this element.
For fix this problem set for input 
padding-left instead margin-left

Reply:
And set for label
```
    width: 100%;
    display: inline-block;